contents/docs/experiments/metrics.mdx

```diff
@@ -0,0 +1,75 @@
+
+## Metric types
+
+There are two types of experiment metrics:
+
+1. **Funnels:** Evaluates using funnel conversions. Read more about how we calculate these in our [funnel statistic](/docs/experiments/funnels-statistics) or [funnel insights](/docs/product-analytics/funnels) docs.
+
+2. **Trends:** Evaluates using aggregate metrics such as total count, average count per user, unique sessions, property values, and more. Trends also let you set the exposure event if you want it to be something other than `$feature_flag_called`. Read more about how we calculate these in our [trend count](/docs/experiments/trends-count-statistics) or [trend continuous](/docs/experiments/trends-continuous-statistics) statistic or [trend insights](/docs/product-analytics/trends/overview) docs.
+
+## Primary and secondary metrics
+
+Each metric can be set as either a **primary** or **secondary** metric. You can set up to 10 primary and secondary metrics per experiment.
+
+**Primary metrics** represent the main goal of your experiment. They directly measure if your hypothesis was successful and are the key factor in deciding if the test achieved its primary objective.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_01_14_at_10_17_53_2x_928092c2ee.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_01_14_at_10_17_22_2x_e0b179b143.png"
+    alt="Primary metrics in PostHog experiments"
+    classes="rounded"
+/>
+
+**Secondary metrics** provide more context for your experiment. They can help you understand the impact of your primary metric and ensure your experiment didn't have negative side effects.
+
+Once added and data starts being collected, we show the results as a credible interval bar chart along with details about significance, win probability, and delta between variants.
+
+### How to read credible interval bar charts
+
+Each bar shows how a variant is performing compared to the control (the gray bar) for the metric, using a 95% credible interval. That means there's a 95% chance the true difference for that variant falls within this range. The vertical "0%" line is your baseline:
+
+- To the right (green): The metric is higher (an improvement).
+- To the left (red): The metric is lower (a decrease).
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_01_14_at_11_19_24_2x_3cadbb532c.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_01_14_at_11_19_43_2x_730755c2c5.png"
+    alt="Credible interval bar chart"
+    classes="rounded"
+/>
+
+The width of the bar represents uncertainty. A narrower bar means we're more confident in that result, while a wider bar means it could shift either way.
+
+The control (baseline) is always shown in gray. Other bars will be green or red—or even a mix—depending on whether the change is positive or negative.
+
+## Shared metrics
```

Comment:
What do you think about putting "Shared metrics" on a separate page so it's visible in the navigation? Not a strong opinion, though.

Reply:
There is too little here at the moment I think. If/when there are some more shared metrics features, we can split it out.

Reply:
Sounds good